core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts

```diff
@@ -235,4 +235,6 @@ export default {
     'Art and Design',
   ],
   DEFAULT_SPEC_TIMEOUT_MSECS: 300000,
+  // Video Recording Path in relative to root of oppia folder.
+  TEST_VIDEO_DIR: path.resolve('../test-recordings/acceptance-tests'),
```

Comment:
Does this create a /test-recordings folder if it doesn't exist yet?

Maybe we can rename to /oppia_full_stack_test_video_recordings/acceptance and /oppia_full_stack_test_video_recordings/e2e. (The video recording folder is a sibling to the oppia/ folder, right?)

Reply:
@seanlip Yes, it will create folder if it doesn't exist. Also, will rename it. Thanks!

Reply:
done!